config/database.php

```diff
@@ -28,6 +28,19 @@
                 PDO::MYSQL_ATTR_SSL_VERIFY_SERVER_CERT => env('MYSQL_ATTR_SSL_VERIFY_SERVER_CERT', true),
             ]) : [],
         ],
+            'pgsql' => [
+                'driver' => 'pgsql',
+                'host' => env('DB_HOST', 'localhost'),
+                'port' => env('DB_PORT', '5432'),
+                'database' => env('DB_DATABASE', 'panel'),
+                'username' => env('DB_USERNAME', 'pterodactyl'),
```

Comment:
```suggestion
                'username' => env('DB_USERNAME', 'pelican'),
```

Reply:
lol yeah i changed mine after i did that